modules/layers/src/path-layer/path-layer.js

```diff
@@ -229,19 +229,19 @@ export default class PathLayer extends Layer {
     const SEGMENT_INDICES = [
       // start corner
```

Comment:
Nit: Use prettier-disable comments to avoid such annoying multi-line formatting.